client/src/App.js

```diff
@@ -4,24 +4,22 @@ import About from "./pages/About";
 import Donations from "./Donations";
 import ThankYou from "./pages/ThankYou";
 import Home from "./pages/Home";
-import About from "./pages/About";
```

Comment:
Looks like we use this on line 18. Does the /about URL work?

Reply:
yes. you may test it from this branch. let me know if it does not, please